.github/workflows/ticdc_integration.yaml

```diff
@@ -205,3 +205,65 @@ jobs:
         if: ${{ always() }}
         run: |
           $GITHUB_WORKSPACE/scripts/canal/canal-local-test.sh down
+
+  CanalJson-with-watermark-integration:
```

Comment:
Is it necessary to make `CanalJson-with-watermark-integration` a new Action, in addition to the existing one? Will enabling watermarks on the existing Canal Action suffice?

Reply:
I will add another action for flink test, I would create a new workflow for extension test.